src/System.Private.CoreLib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -1036,14 +1057,21 @@
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.CreateFile2.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.CREATEFILE2_EXTENDED_PARAMETERS.cs" />
   </ItemGroup>
+  <ItemGroup Condition="($(TargetsWindows) and '$(EnableWinRT)' != 'true') or '$(FeaturePal)'=='true'">
```

Comment:
```suggestion
  <ItemGroup Condition="$(TargetsWindows) or '$(FeaturePal)'=='true'">
```

to be in sync with https://github.com/dotnet/coreclr/pull/22106/files#r251197096

Reply:
The Win32 in the name won't make sense once you do this since the file will be applicable to all Windows.

Maybe `Environment.Variables.Windows.cs` would look better?